gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/avro/HiveAvroSerDeManager.java

```diff
@@ -56,6 +56,10 @@
   public static final boolean DEFAULT_USE_SCHEMA_FILE = false;
   public static final String SCHEMA_FILE_NAME = "schema.file.name";
   public static final String DEFAULT_SCHEMA_FILE_NAME = "_schema.avsc";
+  public static final String SCHEMA_TEMP_FILE_NAME = "schema.temp.file.name";
```

Comment:
dots between words .. but I see you tried to follow naming convention of other config keys. may be we can break convention to do the right thing?

Reply:
This is very similar to the existing `schema.file.name`, so I thought it would be confusing to have `schema.tempFileName`, but leave the other one as `schema.file.name`.